installing/installing_ibm_power/installing-ibm-power.adoc

```diff
@@ -110,12 +110,6 @@ include::modules/installation-registry-storage-non-production.adoc[leveloffset=+
 
 include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
 
-include::modules/cluster-telemetry.adoc[leveloffset=+1]
```

Comment:
Add this again and in addition insert a new line [role="_additional-resources"] before line 115. 

Telemetry only needs to be removed for restricted install  installing\installing_ibm_power\installing-restricted-networks-ibm-power.adoc